openms-wiki/users-quickstart-guide.md

```diff
@@ -0,0 +1,8 @@
+Users Quick Start Guide
+=======================
+
+Download the current stable version of OpenMS from the [OpenMS download site](http://open-ms.de/downloads) or via the [OpenMS docker containers]().
+
+Familiarise yourself with OpenMS by reading [Röst et al, OpenMS: a flexible open-source software platform for mass spectrometry data analysis. Nat Methods. 2016](https://www.ncbi.nlm.nih.gov/pubmed/27575624) as well as [Getting Started](http://www.openms.de/getting-started) on openms.de.
```

Comment:
use: openms.org.

Reply:
openms.org doesn't work for me

Reply:
oh it will redirect, it points to openms.de.